applications/Parallel.md

```diff
@@ -129,38 +129,38 @@ The lending protocol was inspired by compound protocol and our blockchain soluti
-|     4. | User Testing               | We will conduct user testing to improve our product's UX and UI, to ensure that the borrowing and lending functionalities are intuitive. Initially, we will conduct qualitative user testing by observing 10-15 users use the v1 platform and provide a summary of the findings and improvements made based on insights.                                                                            |
+|    1.a | Substrate module: Loans pallet | Loans pallet will be implemented as a cross-asset lending protocol which offers lending and borrowing by using floating rate model.                                                                                     |
+|    1.b | Substrate module: Liquidation pallet | We will implement a liquidation solution build with Substrate Offchain Worker, which will calculates the health factor of each borrower's account and send a liquidation transaction automatically.                                                                                     |
+|    1.c | Substrate module: Price aggregation oracle   |  We will need to create a on-chain price aggregation pallet for the oracle prices from multiple source that is queried by off-chain workers.                                                                            |
+|    2.a | Integration with front-end | We will integrate our existing front end to the finalized substrate backend                                                                                                                                                   |
+|    2.b | Article/Tutorial           | We will create an article and a video demo that explains how users can start using the platform in lending and borrowing for DOT or KSM                                                                                      |
+|    3. | Docker                     | We will provide a dockerfile to demonstrate the lending functionality                                                                                                                                                |
+|    4. | Testing and Documentation  | While we develop our module, we will ensure that our code has significant test coverage (>80%) to ensure quality and provide explanations on how the functions work for the community to review, as well as craft test guides |
+|    5. | User Testing               | We will conduct user testing to improve our product's UX and UI, to ensure that the borrowing and lending functionalities are intuitive. Initially, we will conduct qualitative user testing by observing 10-15 users use the v1 platform and provide a summary of the findings and improvements made based on insights.                                                                            |
```

Comment:
We will conduct user testing to improve our product's UI and UX, to ensure that the borrowing and lending functionalities are intuitive. Initially, we will conduct qualitative user testing by observing 10-15 users who will use the v1 platform and provide a summary of the findings and improvements made based on insights.

Reply:
Please address this comment